src/generator-typemanager.cs

```diff
@@ -148,24 +153,51 @@ static Type Lookup (Assembly assembly, string @namespace, string @typename, bool
 
 	public static Type GetUnderlyingNullableType (Type type)
 	{
+#if IKVM
```

Comment:
I see there is some stuff we need to "workaround" using IKVM, this is a nice example (this is more a comment for future me 😉 )